backend/src/main/java/io/castles/core/model/dto/GameDTO.java

```diff
@@ -0,0 +1,24 @@
+package io.castles.core.model.dto;
+
+import io.castles.core.tile.Tile;
+import io.castles.game.Game;
+import lombok.Value;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Value
+public class GameDTO {
+    GameStateDTO gameState;
+    Map<Integer, Map<Integer, Tile>> tiles;
+    List<PlayerDTO> players;
+
+    public static GameDTO from(Game game) {
+        return new GameDTO(
+                GameStateDTO.from(game),
+                game.getGameBoardTileMap(),
```

Comment:
@soerenreichardt is there an elegant way of mapping the inner Tile instances to TileDTOs here?

Reply:
I came up with this solution:
https://github.com/soerenreichardt/CastlesIO/blob/95889ca0e2f15b486e5c041591f3fc43da9629c1/backend/src/main/java/io/castles/core/model/dto/GameDTO.java#L20-L24

Its more readable than i initially thought. What do you think?

Reply:
The only thing that comes to my mind is not storing it a doubly nested map but have a class called `Position` which takes the x and y coordinate and only store a `Map<Position, TileDTO>`. But otherwise this looks fine

Reply:
depending on the way i process the tiles on the frontend the maps could be handy. I leave it as is for now.